.github/workflows/linux_unit_tests_with_latest_deps.yaml

```diff
@@ -14,17 +14,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - python_version: "3.9"
-            command: 'git-test-automl'
-          - python_version: "3.9"
-            command: 'git-test-prophet'
-          - python_version: "3.9"
-            command: 'git-test-modelunderstanding'
-          - python_version: "3.9"
-            command: 'git-test-other'
-          - python_version: "3.9"
-            command: 'git-test-parallel'
+        python_version: ['3.9']
```

Comment:
Should we also be running tests with 3.10 and 3.11 here?

Reply:
Looks like adding 3.11 test fails `unit git-test-automl tests (pull_request)`.

I think I'll just add 3.10 for now, and file a ticket to add 3.11